src/content/rules/prioritize-official-translations.mdx

```diff
@@ -15,5 +15,37 @@ rationale:
 status: Active
 rule_context: Romanization
 mikumod_support: "False"
+excerpt: The official romanization and/or translation should be prioritized even if it's against VocaDB's [romanization guidelines](/docs/romanization-guidelines).
```

Comment:
I know this is a technical PR and not a docs/style PR, but please change this to "Prioritize the official..."

Reply:
This excerpt is retained from the beginning. My initial suggestion would be...

> The official romanization and/or translation should be prioritized.

I would try mix your idea with my into this. Feels too short but should be fine? LMK is this is not satifactory.

> Prioritize the official romanization and/or translation.

PS. My other idea would be this:

> Prioritize the official romanization and/or translation even if it is lesser known or breaks VocaDB romanization guidelines.